extensions/defaults/timeouts/timeouts.go

```diff
@@ -0,0 +1,40 @@
+package timeouts
+
+import "time"
+
+func TwoMinuteWatch() *int64 {
+	timeout := int64(60 * 2)
+	return &timeout
+}
+
+func ThreeMinuteWatch() *int64 {
+	timeout := int64(60 * 3)
+	return &timeout
+}
+
+func TenMinuteWatch() *int64 {
+	timeout := int64(60 * 10)
+	return &timeout
+}
+
+func TwentyMinuteWatch() *int64 {
+	timeout := int64(60 * 20)
+	return &timeout
+}
+
+func ThirtyMinuteWatch() *int64 {
+	timeout := int64(60 * 30)
+	return &timeout
+}
```

Comment:
Same for this one too, if we add a new timeout duration, isn't this file going to get longer for the same const?

```suggestion
func NewTimeoutDuration(mins int) *int64 {
	timeout := int64(60 * mins)
	return &timeout
}
```

Reply:
1. I like the idea your proposing so i'll implement it
2. Would it make sense for me to create a second constant in addition to NewTimeoutDuration that is the default that we use across the majority of our framework. The reason I mention this is because the majority of all of our timeoutDurations are the ThirtyMinuteWatch, and one of the major selling points of a defaults file is to allow us to update the constant in all files without having to go and update each file individually i.e update the input of NewTimeoutDuration in all files.

Reply:
For sure that makes sense to me, I agree, for your second point; Do you mean something like this:

```
type Duration int

const (
  ThirtyMinutes Duration = 30
  FifteenMinutes Duration = 15
)

func NewTimeoutDuration(mins Duration) *int64 {
	timeout := int64(60 * int(mins))
	return &timeout
}
```

Reply:
Yep, that would be a good way of handling it. I'll get that implemented.